weedcoco/importers/voc.py

```diff
@@ -4,6 +4,18 @@
 import argparse
 import json
 
+try:
+    from pycocotools.coco import COCO
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import time
+    import cv2
+    import os
+
+    visualisation = True
```

Comment:
We should make this a command-line arg on all importers if we wish it to be most useful.